tutorials/vr/vr_starter_tutorial.rst

```diff
@@ -0,0 +1,1211 @@
+          
+          The audio used are from several different sources, all **downloaded from the Sonnis #GameAudioGDC Bundle**. The license for the sound effects is included as a PDF
+          and the folders where the audio is stored have the same name as the sources whom they came from. The files were downloaded from https://GameSounds.xyz
+          
+          The **OpenVR addon** is created by BastiaanOliji, and is released under the MIT license, and can be found both on the Godot Asset Library, and here on
+          Github: https://github.com/GodotVR/godot-openvr-asset
+          
+          Everything else is original and created solely for this tutorial by TwistedTwigleg. They are released under the MIT license, so feel free to use however you see fit!
+
+.. tip:: You can find the finished project at the bottom of this page
+
+Getting everything ready
+------------------------
+Launch Godot and open up the project included in the starter assets.
+
+.. note:: While these assets are not necessarily required to use the scripts provided in this tutorial,
+          they will make the tutorial much easier to follow as there are several pre-setup scenes we
+          will be using throughout the tutorial series.
+
+First, you may notice there is already quite a bit set up. This includes: A pre-built level, and several instanced scenes placed around,
+some background music, and several GUI related :ref:`MeshInstances <class_MeshInstance>`. You may notice that the GUI related meshes already have a script
+attached to them, and this is simply used to show whatever is inside the :ref:`Viewport <class_Viewport>` on the mesh. Feel free to take a look if you want, but
+this tutorial will not be going over how to use the :ref:`Viewport <class_Viewport>` nodes for making 3D GUI :ref:`MeshInstance <class_MeshInstance>`s. 
+
+The other thing to notice before we jump in to writing the code is how the :ref:`ARVROrigin <class_ARVROrigin>` node works. How it works is kind of hard to explain,
+especially if you have never used VR before, but here is the gist of it:
+The :ref:`ARVROrigin <class_ARVROrigin>` node is the center point of the room. If there is no room scale tracking, then the :ref:`ARVROrigin <class_ARVROrigin>` will
+be directly below the player, but if there is room scale tracking, then the :ref:`ARVROrigin <class_ARVROrigin>` will be the center of the tracked room.
+
+.. note:: This is a bit of a simplification, and honestly, I do not know enough about the various different VR headsets and how they work to give a more detailed
+          and complete explanation. The simple way is to look at it like this: The :ref:`ARVROrigin <class_ARVROrigin>` is the center of the VR world. If there is
+          room tracking, the player can move away from the center point, the :ref:`ARVROrigin <class_ARVROrigin>` node, but only as far as the room scaling tracks.
+
```

Comment:
Indeed, hard to explain this much better without just confusing the reader. OpenVR for the most part ensures that the origin is akin to the center of the world and will adjust tracking. So while Oculus tracks from Cameras, it still has an origin point on the floor at the center of the room. 

This will only start to be different if you use something other then OpenVR.